### PR TITLE
Add MATLAB implementation

### PR DESCRIPTION
**Pull request by sergiolibe**
Is a simplest implementation in MATLAB, hope you like it!

### CODE REVIEW

File: MATLAB/stalinsort.m [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function [orderedList] = stalinsort(list)
+
+orderedList = [list(1)];current=list(1);
+for(i=2:length(list))if(list(i)>current)
```
Comment (gustavo-depaula) [inline MATLAB/stalinsort.m]: what happens to the input `1 3 2 4 4 6`?

Comment (sergiolibe) [reply MATLAB/stalinsort.m]: It works with that input [1 3 2 4 4 6].
Or any of this examples
a) [1, 2, 3, 4, 5, 6, 7, 8, 9]
b) [9, 8, 7, 6, 5, 4, 3, 2, 1]
c) [1, 3, 2, 5, 4, 7, 6, 9, 8]
d) [1, 5, 2, 4]

with or without the commas but the square brackets are necessary

Comment (gustavo-depaula) [reply MATLAB/stalinsort.m]: but what is the output with repeated numbers?

Comment (sergiolibe) [reply MATLAB/stalinsort.m]: Repeated numbers are ignored (duplicates), just saving one of them.
imagen

Comment (gustavo-depaula) [reply MATLAB/stalinsort.m]: Ok! We need to keep the repeated numbers

Comment (gustavo-depaula) [reply MATLAB/stalinsort.m:4]: @sergiolibe can you fix this issue so we can merge the PR?

Comment (elliotwutingfeng) [reply MATLAB/stalinsort.m:4]: Here's my fixed implementation. Now works on empty arrays too. Tested on MATLAB R2022a.

```matlab
function [orderedList] = stalinsort(list)
if isempty(list)
    orderedList = [];
else
    orderedList = [list(1)];
    current = list(1);
    for i=2:length(list)
        if(list(i) >= current)
            current = list(i);
            orderedList = [orderedList,current];
        end
    end
end
```

File: MATLAB/stalinsort.m [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+	current=list(i);
+	orderedList=[orderedList,current];
+end end
```
Comment (elliotwutingfeng) [inline MATLAB/stalinsort.m:7]:
```suggestion
function [orderedList] = stalinsort(list)
if isempty(list)
    orderedList = [];
else
    orderedList = [list(1)];
    current = list(1);
    for i=2:length(list)
        if(list(i) >= current)
            current = list(i);
            orderedList = [orderedList,current];
        end
    end
end
```

**Comment by elliotwutingfeng** [comment]
😭

**Pull request closed without merging**